src/AltinnCore/Common/Services/Interfaces/ISourceControl.cs

```diff
@@ -12,7 +12,7 @@ namespace AltinnCore.Common.Services.Interfaces
     public interface ISourceControl
     {
         /// <summary>
-        /// Clone service repository to local repo
+        /// Clone app repository to local repo
```

Comment:
service

Reply:
not sure I agree here, @dskogan . 
It is the gitea repository for the application, so why not refere to it as the app repository?

Reply:
I don't quite understand why we should use service here instead of app? Or did you mean to comment on line 10 "Interface for source control functionality" -> "Interface for source control service"?